src/components/navigation/unread-notifications.tsx

```diff
@@ -0,0 +1,23 @@
+import { useGraphqlSwr } from '@/api/use-graphql-swr'
+
+export function UnreadNotifications() {
+  const { data } = useGraphqlSwr<{
+    notifications: {
+      totalCount: number
+    }
+  }>({
+    query: `
+      {
+        notifications(unread: true) {
+          totalCount
+        }
+      }
+    `,
+    config: {
+      refreshInterval: 60 * 1000, // seconds
+    },
+  })
+  return data ? (
+    <>You have {data.notifications.totalCount} unread notifications</>
```

Comment:
Could be a badge in the menu bar that shows how many unread notifications you have.